Documentation/ceph-filesystem-crd.md

```diff
@@ -51,6 +53,41 @@ spec:
     #    memory: "1024Mi"
 ```
 
+(These definitions can also be found in the [`filesystem.yaml`](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/filesystem.yaml) file)
+
+### Erasure Coded
+
+If you want to use erasure coded pool with filesystem, your OSDs must use `bluestore` as their `storeType`.
+Additionally erasure coded can only be used as a data pool and not as a metadata pool. The metadata pool must still be a replicated pool.
+
+To make the metadata of the filesystem in this example reliable you need to increase the `metadataPool` replicated size if you have more than one OSD.
+For erasure coded to make sense, you need **at least three OSDs for the below `dataPools` config** to work.
+
+```yaml
+apiVersion: ceph.rook.io/v1beta1
+kind: Filesystem
+metadata:
+  name: myfs-ec
+  namespace: rook-ceph
+spec:
+  metadataPool:
+    replicated:
+      size: 1
```

Comment:
since the erasure-coded data pool can tolerate one loss, seems like we should set the metadata pool replication to at least 2 so it can also tolerate a loss.